css/style.css

```diff
@@ -0,0 +1,715 @@
+@import url('https://fonts.googleapis.com/css2?family=Oswald:wght@200;300;400;500;600;700&display=swap');
+
+body {
+  font-family: 'Oswald', sans-serif;
+  font-style: normal;
+  font-weight: 100;
+  max-width: 1920px;
+  height: 100px;
+  margin: 0;
+  padding: 0;
+  color: #333;
+  overflow-x: hidden;
+}
+
+.mobile-img {
+  display: none;
+}
+
+.menumobile {
+  display: none;
+}
+
+.top-header {
+  display: flex;
+  width: 100%;
+  position: absolute;
+  max-width: 1920px;
+  height: 36px;
+  justify-content: flex-end;
+  align-items: center;
+  box-sizing: border-box;
+  background-color: #272a31;
+}
+
+.header-content {
+  display: flex;
+  position: absolute;
+  width: 45%;
+  align-content: center;
+}
+
+.header-list {
+  list-style-type: none;
+  display: flex;
+  justify-content: flex-end;
+}
+
+.header-list li {
+  justify-content: space-between;
+  margin: 0 20px;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 14px;
+  line-height: 24px;
+  text-align: right;
+  color: #fff;
+}
+
+.navigation {
+  display: flex;
+  width: 100%;
+  position: relative;
+  justify-content: space-evenly;
+  align-items: center;
+  box-sizing: border-box;
+  align-content: center;
+  padding: 10px;
+  box-shadow: 0 10px 18px #888;
+  top: 36px;
+  height: 80px;
+}
+
+.logo,
+.logo-img {
+  vertical-align: middle;
+  display: flex;
+  justify-content: space-between;
+  height: 95%;
+  cursor: pointer;
+}
+
+.menu {
+  display: flex;
+  height: 80px;
+  float: right;
+  color: rgb(0, 0, 0);
+  text-align: right;
+  vertical-align: middle;
+  position: relative;
+  align-content: center;
+  align-items: center;
+}
+
+.menu-nav {
+  list-style-type: none;
+  display: flex;
+  justify-content: flex-start;
+  color: #272a31;
+}
+
+.menu-nav li {
+  color: grey;
+  justify-content: space-between;
+  margin: 0 20px;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 16px;
+  line-height: 24px;
+  text-align: right;
+}
+
+.hero {
+  display: flex;
+  position: relative;
+  text-align: center;
+  top: 56px;
+}
+
+.call-title {
```

Comment:
The original used "image" to do all, I created the texts and the effects, and I was just following the project's instructions as a quote here: "We provide some design guidelines for you to create the website, **but you must personalize the content,** i.e., instead of a conference you can build a website for a concert or for a web development course. **It's very important the you personalize your project** so that you have something unique in your portfolio to share with potential employers during job searching". If I understood well the requirements, it's my call to decide how looks like will be my development.